go/vt/vttablet/tabletserver/planbuilder/testdata/exec_cases.txt

```diff
@@ -339,7 +339,7 @@
     }
   ],
   "FullQuery": "update d set foo = 'foo' where `name` in ('a', 'b') limit :#maxLimit",
-  "WhereClause": "where `name` in ('a', 'b')"
+  "WhereClause": " where `name` in ('a', 'b')"
```

Comment:
These cases change since they use the JSON serialization mentioned earlier and a side-effect of `TruncateForUI` was that it also trimmed whitespaces. We now dropped that so the original text is still here. It has no functional change though.